torchvision/transforms/transforms.py

```diff
@@ -434,17 +434,17 @@ def __init__(self, size, padding=None, pad_if_needed=False, fill=0, padding_mode
         self.padding_mode = padding_mode
 
     @staticmethod
-    def get_params(img, output_size):
+    def get_params(w, h, output_size):
```

Comment:
I have some reservations with respect to changing the API of the existing transforms, but I wonder how often this particular one is used externally.

Should we issue a warning maybe (cc @fmassa)?

Reply:
yeah, we should not be doing a BC-breaking change here. There are ways of achieving the same thing without breaking BC, see for example https://github.com/pytorch/vision/pull/1104/files#diff-fc1f220b470714d05cf3ea6acf9fed59R34